app/routes/pub-sub.js

```diff
@@ -25,25 +25,17 @@ router.post('/receive', express.json(), async (req, res) => {
       Buffer.from(req.body.message.data, 'base64').toString('utf-8');
   // Parse the above string into an object, and destructure the id from it
   const { id } = JSON.parse(message);
```

Comment:
We can probably remove this parsing now right, since we're not using the id for anything in the subsequent logic?